### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by ChalankaKodikara** [ChalankaKodikara/AgrifirstFYP#1] [Q&A] [answered]
How to create a JWT token ?

**Comment by Thameera08** [comment, marked as answer]
import jwt
import datetime

# Define your secret key and data
secret_key = "your_secret_key"
payload = {
    "user_id": 123,
    "username": "exampleuser",
    "exp": datetime.datetime.utcnow() + datetime.timedelta(hours=1)  # Token expiration
}

# Encode the token
token = jwt.encode(payload, secret_key, algorithm="HS256")
print("JWT Token:", token)
